### PR TITLE
Switch project to Gradle version catalog

### PR DESCRIPTION
**Pull request by afalhambra-hivemq**

### CODE REVIEW

File: tests-hivemq-operator/build.gradle.kts [hunk 1]
```diff
@@ -19,24 +19,31 @@ repositories {
...
-    testImplementation("ch.qos.logback:logback-core:${property("logback.version")}")
-    testImplementation("org.slf4j:slf4j-api:${property("slf4j.version")}")
-    testImplementation("org.testcontainers:hivemq:${property("testcontainers.version")}")
+    // Testing
+    testImplementation(libs.assertj)
+    testImplementation(libs.awaitility)
+
+    // JUnit
+    testImplementation(libs.junit.jupiter)
+    testImplementation(libs.junit.jupiter.api)
```
Comment (Donnerbart) [inline tests-hivemq-operator/build.gradle.kts:28]: I'm pretty sure we don't need the API dependency if we have `junit-jupiter` as dependency. That should be a transitive dependency.

Comment (afalhambra-hivemq) [reply tests-hivemq-operator/build.gradle.kts]: good one, thanks

**Review by Donnerbart — APPROVED** [review]
Awesome cleanup of the version management!

**Comment by afalhambra-hivemq** [comment]
Re-triggered manually the CI job, it seems the report status was not sent back or the job was not triggered at all.

**Pull request merged**